common/models/workspace.js

```diff
@@ -50,6 +50,20 @@ module.exports = function(Workspace) {
      */
 
     /**
+     * Get list of available loopback Versions.
+     *
+     * @return {Object} availableLBVersions
+     */
+
+    Workspace.getAvailableLBVersions = function() {
+      var availableLBVersions = {
+        '2.x': { description: 'stable' },
+        '3.x': { description: 'pre-release, alpha quality' },
+      };
+      return availableLBVersions;
+    };
```

Comment:
We need to make this method async, so that it can be exposed via REST API for consumption by APIC Editor (no need to add remoting metadata right now though).

```js
Workspace.getAvailableLBVersions = function(cb) {
  // ...
  cb(null, availableLBVersions);
};
```

See e.g. `getAvailableTemplates` which is already async.